### PR TITLE
Add locked app_oflline file test and remove retrying client

### PR DESCRIPTION
**Pull request by pakrym**
Everything should work without retries.

### CODE REVIEW

File: test/Common.FunctionalTests/AppOfflineTests.cs [hunk 1]
```diff
@@ -42,6 +42,22 @@ public async Task AppOfflineDroppedWhileSiteIsDown_SiteReturns503(HostingModel h
             DeletePublishOutput(deploymentResult);
         }
 
+        [ConditionalTheory]
+        [InlineData(HostingModel.InProcess)]
+        [InlineData(HostingModel.OutOfProcess)]
+        public async Task LockedAppOfflineDroppedWhileSiteIsDown_SiteReturns503(HostingModel hostingModel)
```
[the remark this answers is not in the document]
Comment (pakrym) [reply test/Common.FunctionalTests/AppOfflineTests.cs]: While file is locked we expect server to return 503 with empty content

[the remark this answers is not in the document]
Comment (pakrym) [reply test/Common.FunctionalTests/AppOfflineTests.cs]: I guess I should have written something to file to illustrate that.

[the remark this answers is not in the document]
Comment (mikeharder) [reply test/Common.FunctionalTests/AppOfflineTests.cs]: Does the server check the file exactly once, and cache the result (content or locked) until the file is deleted?  Or does it update in real-time as the file is locked and unlocked?  In this test, after you unlock the file, would the server continue to return empty content, or would it return the file content?

[the remark this answers is not in the document]
Comment (pakrym) [reply test/Common.FunctionalTests/AppOfflineTests.cs]: It re-reads file on every poll (every 200ms). Might be a nice test to add.

**Pull request merged**